File: 08_기본수학_2/Problem_03.cpp
/* 08단계 [ 기본수학 2 ] */
// 제목 : 소인수분해
// 문제 번호 : 11653

#include <iostream>
using namespace std;

int main(void)
{
    int N, i = 2;
    cin >> N;
    while (N != 1)
    {
        if (N % i)
            i++;
        else
        {
            cout << i << endl;
            N /= i;
        }
    }
}

File: Python/06_문자열/Problem_03.py
# int main()
# {
#     string s;
#     string alphabet = "abcdefghijklmnopqrstuvwxyz";
#     cin >> s;
#     for (int i = 0; i < alphabet.length(); i++)
#         cout << (int)s.find(alphabet[i]) << " ";
#     return 0;
# }


s = input()
alphabet = "abcdefghijklmnopqrstuvwxyz"
for i in alphabet:
    if i in s:
        print(s.find(alphabet[i]), end=' ')